lib/bootvar.ml

```diff
@@ -35,10 +35,10 @@ let get_cmd_line () =
 
 let create () = 
   get_cmd_line () >>= fun cmd_line ->
-  let entries = Re_str.(split (regexp_string " ") cmd_line) in
```

Comment:
I understand this already used to be the case but the user still has to pray he will never have to put a space or an `=` in a parameter value. Do you want users to hate your software ? 

See #18 and the `parse_env` example [here](http://erratique.ch/software/astring/doc/Astring#examples).